lib/widgets/core/about.dart

```diff
@@ -22,99 +22,164 @@ import 'package:provider/provider.dart';
 import 'package:wger/helpers/misc.dart';
 import 'package:wger/providers/auth.dart';
 
-class WgerAboutListTile extends StatelessWidget {
+class AboutPage extends StatefulWidget {
+  static String routeName = '/AboutPage';
+  const AboutPage({Key? key}) : super(key: key);
+
+  @override
+  State<AboutPage> createState() => _AboutPageState();
+}
+
+class _AboutPageState extends State<AboutPage> {
   @override
   Widget build(BuildContext context) {
+    final deviceSize = MediaQuery.of(context).size;
     final authProvider = Provider.of<AuthProvider>(context, listen: false);
 
-    return AboutListTile(
-      //dense: true,
-      icon: const Icon(Icons.info),
-      applicationName: 'wger',
-      applicationVersion: 'App: ${authProvider.applicationVersion!.version}\n'
-          'Server: ${authProvider.serverVersion}',
-      applicationLegalese: '\u{a9} 2020 - 2021 contributors',
-      applicationIcon: Padding(
-        padding: const EdgeInsets.only(top: 10),
-        child: Image.asset(
-          'assets/images/logo.png',
-          width: 60,
-        ),
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('About wger'),
```

Comment:
Can you mark this string as translatable? Then this can be merged

Just add it to app_en.arb and use here something like AppLocalizations.of(context).aboutPageTitle

Reply:
Okay I'll do that and update the PR.